pkg/network/ovn_kubernetes.go

```diff
@@ -141,6 +141,9 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["RoutableMTU"] = nil
 	data.Data["V4JoinSubnet"] = c.V4InternalSubnet
 	data.Data["V6JoinSubnet"] = c.V6InternalSubnet
+	data.Data["V4InternalMasqueradeSubnet"] = c.V4InternalMasqueradeSubnet
```

Comment:
this and the following ones will be `c.GatewayConfig.V4InternalMasqueradeSubnet`

Reply:
why would these be `c.GatewayConfig.V4InternalMasqueradeSubnet` when the join subnet above in the same config is not?